src/manage_sql_tickets.c

```diff
@@ -157,7 +132,14 @@ ticket_status_name (ticket_status_t status)
    { "host", NULL, KEYWORD_TYPE_STRING },                                     \
    { "location", NULL, KEYWORD_TYPE_STRING },                                 \
    { "solution_type", NULL, KEYWORD_TYPE_STRING },                            \
-   { "status", NULL, KEYWORD_TYPE_STRING },                                   \
+   { "(CASE status"                                                           \
+     " WHEN 0 THEN 'Open'"                                                    \
```

Comment:
```suggestion
     " WHEN " TICKET_STATUS_OPEN " THEN 'Open'"                                                    \
```

Would this work?  Would be neater.

Reply:
I don't think this will work because `TICKET_STATUS_OPEN` etc. are part of an enum, not preprocessor constants, so they cannot be used like string literals.

A way around this would be defining a function where we can insert the values into the format string of the sql() call. 
Unless there are other place where we could use the function, I don't think this is worth the effort, though.

Reply:
Oh right, of course.

Another option would be to change the enum to definitions, but no big deal.